psycop/projects/cvd/model_training/preprocessing/bool_to_int.py

```diff
@@ -0,0 +1,21 @@
+import polars as pl
+from polars import Boolean
+
+from psycop.common.model_training_v2.config.baseline_registry import BaselineRegistry
+from psycop.common.model_training_v2.trainer.preprocessing.step import (
+    PolarsFrame_T0,
+    PresplitStep,
+)
+
+
+@BaselineRegistry.preprocessing.register("bool_to_int")
+class BoolToInt(PresplitStep):
```

Comment:
Also seems like a fairly general purpose and generally useful preprocessing step. Simple enough to just move to common imo